system/View/Parser.php

```diff
@@ -574,8 +574,8 @@ protected function replaceSingle($pattern, $content, $template, bool $escape = f
 		$pattern = addcslashes($pattern, '$');
 
 		// Flesh out the main pattern from the delimiters and escape the hash
-		// See https://regex101.com/r/1GIHTa/1
-		if (preg_match('/^(#)(.*)(#(m?s)?)$/', $pattern, $parts))
+		// See https://regex101.com/r/IKdUlk/1
```

Comment:
@paulbalandan to update regex101 version, you can use "Update Regex" button instead of "Save Regex" so the new url generated will be increased number /2 /3, etc, eg: https://regex101.com/r/1GIHTa/2

ref https://twitter.com/samsonasik/status/1355829852138430468